doc/Installation/Installing-LibreNMS.md

```diff
@@ -29,3 +31,5 @@ If the above are missing or incomplete then you can try our old docs below.
 [Debian/Ubuntu Lighttpd](http://docs.librenms.org/Installation/Installation-Ubuntu-1404-Lighttpd/)
 
 [Debian/Ubuntu Nginx](http://docs.librenms.org/Installation/Installation-Ubuntu-1404-Nginx/)
+
+[More]: https://community.librenms.org/t/change-to-php-minimum-version-5-6/2542
```

Comment:
Possibly unnecessary if we just are posting the minimum version as 5.6.4.  We'll advertise this link on social media.